### DISCUSSION TITLE
求你展开这个循环吧 – ChloroBlog

### DISCUSSION THREAD

**Discussion by Chlorie** [Chlorie/ChloroBlog#19] [Announcements]
https://chlorie.github.io/ChloroBlog/posts/2019-07-24/static-for.html 

 能鸽善鹉者的领域

**Comment by c0nstexpr** [comment]
“为时过早的优化是编程中所有（至少大部分）邪恶的起源”。感觉自己现在天天做这个事情……又不想着加新功能

**Comment by Chlorie** [comment]
> @BlurringShadow
> “为时过早的优化是编程中所有（至少大部分）邪恶的起源”。感觉自己现在天天做这个事情……又不想着加新功能

我就是这样的人，不然就没这篇文了（
